signatures/Picnic/Picnic-UR/bench/optimized-C_L1-UR_platformA.yaml

```diff
@@ -0,0 +1,6 @@
+platform: Intel Core i7-4790 3.60GHZ
+timings:
+  unit: cycles
+  gen: 14007
+  enc|sign: 12552077
+  dec|vrfy: 10344492
```

Comment:
Did you not include the memory numbers for a reason or did you just forget to put them?

Reply:
(applies to a few other places as well)

Reply:
> I noticed that there have been updates to picnic, now there is no more picnic2 but picnic3, see https://github.com/microsoft/Picnic/tree/master/spec
> So it seems you were using an old version of the paper? What did you use?
> I've only checked the Picnic-FS and -UR variants for now.
> 
> I'd also like to harmonize with the other schemes by renaming PlatformX to speaking names. I'd suggest Haswell, ARMv8 and Skylake as names, what do you think? If this'd be a lot of work for you I can do it using some shell magic :)

You mean Xeon, Intel, ARM? page 85 in v2.2 . Btw I always use the version of spec submitted to NIST.

Reply:
Haswell/ARMv8/Skylake are the names that the other schemes typically use so I thought that's the best way to name them. I've looked up the processor names and it seems A is a haswell and C is Skylake. Correct me if I'm wrong though :)

Xeon and Intel would not be good names I'd say because Xeon is also an intel.  I chose Skylake because that's what the other schemes used, but I don't really care.

Reply:
yes tha makes sense then